ui/src/app/views/concept-homepage/component.ts

```diff
@@ -223,4 +242,8 @@ export class ConceptHomepageComponent implements OnInit {
       this.conceptSelected = true;
     }
   }
+
+  get noConceptsConstant() {
+    return 'No concepts found for domain \'' + this.selectedDomain.name + '\' this search.';
```

Comment:
can this be done more concisely with double quotes?

Reply:
I think it could, but our linter doesn't like double quotes, and sematically I think it makes more sense to have single quotes in the display.